utils/dataset.py

```diff
@@ -1,6 +1,7 @@
 import csv
 import glob
 import os
+import time
```

Comment:
Please remove the time measurement and print out function as it is redundant.

Reply:
Done